fuzz.go

```diff
@@ -263,7 +273,11 @@ func (fc *fuzzerContext) doFuzz(v reflect.Value, flags uint64) {
 	defer func() { fc.curDepth-- }()
 
 	if !v.CanSet() {
-		return
+		if fc.fuzzer.allowUnexportedFields{
+			v = reflect.NewAt(v.Type(), unsafe.Pointer(v.UnsafeAddr())).Elem()
+		}else{
```

Comment:
Please run gofmt

Reply:
Just did gofmt, please check the latest file

Reply:
Just did gofmt, please checkout the latest files